lib/talon/page_controller.ex

```diff
@@ -5,18 +5,30 @@ defmodule Talon.PageController do
       talon = opts[:concern] || raise("concern option required")
       plug :set_concern, talon: talon
 
-      @spec page(Plug.Conn.t, Map.t) :: Plug.Conn.t
-      def page(conn, params) do
-        page = Map.get(params, "page", "dashboard")
-        render(conn, "#{page}.html")
+
+      @spec index(Plug.Conn.t, Map.t) :: Plug.Conn.t
+      def index(conn, params) do
```

Comment:
My thought was the index explicitly handled the scenario of "/". By breaking out, we provide a convenient hook in the controller to override to the policy of using the default_page, which in turn uses the first page defined. Of course, the user could also override the concern but this might be more intuitive. Thoughts?